modules/node-network-configuration-policy-file.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-observing-node-network-state.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-network-configuration-policy-file.adoc_{context}"]
+= The NodeNetworkConfigurationPolicy manifest file
+
+A `NodeNetworkConfigurationPolicy` (NNCP) manifest file defines policies that the Cluster Network 
+Operator (CNO) uses to configure networking for nodes that exist in an {product-title} cluster. 
+
+After you apply a node network policy to a node, the CNO creates an interface on the node. A node network policy includes your requested network configuration and the status of execution for the policy on the cluster as a whole.
+
+You can create an NNCP by using either the {oc-first} or the {product-title} web console. As a postinstallation task you can create an NNCP or edit an existing NNCP.
+
+[NOTE]
+====
+Before you create an NNCP, ensure that you read the "Example policy configurations for different interfaces" document.
+====
+
+If you want to delete an NNCP, you can use the `oc delete nncp` command to complete this action. However, this command does not delete any created objects, such as a bridge interface. Deleting the node network policy that added an interface to a node does not change the configuration of the policy on the node. Similarly, removing an interface does not delete the policy.
+
+To effectively delete the NNCP, the node network policy, and any created interfaces, you must complete the following actions:
+
+. Edit the NNCP and remove all interface details from the file.
+. Add `state: absent` under the `dns-resolver.config` section of the NNCP.
```

Comment:
Change "dns-resolver.config" to "interface.state" -- because in this section we always say "[...] added an interface", "[...] removing an interface" so let's keep the example about the interface

Reply:
Updates. I see the YAML config in the doc states "interfaces.state" so I updated as per this example. 

![Screenshot from 2025-01-21 12-47-38](https://github.com/user-attachments/assets/95913c84-9cfc-4204-8703-b84f9217d53f)